src/components/generic/Badge/Badge.scss

```diff
@@ -0,0 +1,35 @@
+.wxc-badge {
+  position: relative;

```

Comment:
position, display, text-align, decoration, justify, align, margin, line-height, are not needed here